src/main/java/org/opensearch/knn/index/codec/KNN990Codec/NativeEngines990KnnVectorsWriter.java

```diff
@@ -187,85 +187,28 @@ private <T> KNNVectorValues<T> getKNNVectorValuesForMerge(
         switch (fieldInfo.getVectorEncoding()) {
             case FLOAT32:
                 FloatVectorValues mergedFloats = MergedVectorValues.mergeFloatVectorValues(fieldInfo, mergeState);
-                return (KNNVectorValues<T>) KNNVectorValuesFactory.getVectorValues(vectorDataType, mergedFloats);
+                return getVectorValues(vectorDataType, mergedFloats);
             case BYTE:
                 ByteVectorValues mergedBytes = MergedVectorValues.mergeByteVectorValues(fieldInfo, mergeState);
-                return (KNNVectorValues<T>) KNNVectorValuesFactory.getVectorValues(vectorDataType, mergedBytes);
+                return getVectorValues(vectorDataType, mergedBytes);
```

Comment:
Just a thought not a comment, is there a reason for using static import over 
```
KNNVectorValuesFactory.getVectorValues(vectorDataType, mergedFloats)
```
I know it doesn't matter and more a coding style and personal preference, it just feel easy to walk through codes in browser if we do this:
```
KNNVectorValuesFactory.getVectorValues(vectorDataType, mergedFloats)
```
over 

```
getVectorValues(vectorDataType, mergedFloats)
```

Reply:
Its just easy on the eyes and helps with verbosity, no other reason. I can change it back if there is preference.

Either way its good to standardize it across the code base irrespective of what's chosen.

Reply:
Sure make sense. No hard preference. Its personal preference. :)